.devcontainer/devcontainer.json

```diff
@@ -42,5 +42,8 @@
         ]
       }
     }
-  }
+  },
+  "mounts": [
```

Comment:
This seems like a personal mount. btw, if you open 2 dev containers but use same disk, you can just access the other files.

Reply:
🤦🏼